Added ability to aim the cue by dragging the mouse

ENiGMA9 opened the pull request:
This solves #62

ENiGMA9 committed:
Added the ability to aim the cue by mouse movement

Moved the logic from mouse click (as it was not useful anymore there) to mouse movement. Also removed a useless line from an older commit.

ENiGMA9 committed:
Updated Ability to aim by drag

This builds upon last commit, I reread the description of the issue and modified accordingly. Now you are required to press the left mouse button.

sleddog commented on index.html:
```diff
@@ -801,7 +803,8 @@
    {
-       var cue_ball = ballList.balls[0];
+       //Obsolete now but I leave it here for easy modifications
+      /* var cue_ball = ballList.balls[0];
```
@ENiGMA9 if this isn't needed feel free to clean up.  Github keeps the history nicely :)

ENiGMA9 committed:
Removed obsolete code

As suggested, by the way, thank you, I learn new things about github doing these :D

ENiGMA9 commented:
Thanks, I did as suggested, now about the click thing, currently works anywhere in the canvas not only on the ghost ball, would you like me to change this?

sleddog commented:
@ENiGMA9 Sure, if you can improve it give it a shot (pun intended).  Looking for ideas to improve how the user aims the next shot.

ENiGMA9 commented:
@sleddog I worked a bit last night but didn't manage easily to pull this off, I meant to commit to this PR but I didn't do it in time and I still have to work on it as it's not functional right now, although I would say it's a bit better like it is now, control-wise? I feel like 8ball pool from miniclip is good on controls, if you want to take inspiration from that.

sleddog commented:
@ENiGMA9 8ball pool from miniclip is rock solid.  If we could build a game that is close to that, then wow!  For now I'm happy with the aiming improvements you added.  If you have more ideas for improvement let me know.  Thanks again.
